packages/govtnz-ds-website/src/docs/basics/themes__examples.md

```diff
@@ -123,3 +132,9 @@
     </Example>
 
 </ExampleContainer>
+
+## Customising a theme
+
+You can customise the Design System theme to follow your brand guidelines by altering the values in the [Sass variables file](https://github.com/GOVTNZ/govtnz-design-system/blob/master/packages/govtnz-ds/build/scss/_settings.scss).
+
+We encourage you to make the look, feel, and usability of elements as consistent as possible with the Design System. And when you customise, make sure your visual elements continue to meet the NZ [Government Web Accessibility Standard](https://www.digital.govt.nz/standards-and-guidance/nz-government-web-standards/web-accessibility-standard-1-1/).
```

Comment:
@holloway checking this is the correct place  for theme content. I ran` yarn compile-markdown ` and it didn't seem to update. I will investigate more :)

Reply:
@Juliet-Brown Nah it's in `packages/govtnz-ds-website/src/pages/basics/themes.tsx` :)